### PR TITLE
Manually bump dependencies

### PR DESCRIPTION
**Pull request by sharathsivakumar**
Here are the dependencies for this week:

- https://github.com/etcd-io/etcd/pull/17897
- https://github.com/etcd-io/etcd/pull/17898
- https://github.com/etcd-io/etcd/pull/17899
- https://github.com/etcd-io/etcd/pull/17900
- https://github.com/etcd-io/etcd/pull/17901
- https://github.com/etcd-io/etcd/pull/17906 --> Bumped
- https://github.com/etcd-io/etcd/pull/17907
- https://github.com/etcd-io/etcd/pull/17908 --> Bumped
- https://github.com/etcd-io/etcd/pull/17909
- https://github.com/etcd-io/etcd/pull/17910 --> Bumped

Please let me know if any changes are required

### CODE REVIEW

**Comment by ivanvc** [comment]
Hi @sharathsivakumar, thanks for the PR. It looks like `go.opentelemetry.io/otel` #17910 is not indirect; it's used by `tests` and `server`. Refer to the following snippet:

```
etcdutl/go.mod
61:     go.opentelemetry.io/otel v1.25.0 // indirect

tests/go.mod
39:     go.opentelemetry.io/otel v1.25.0

server/go.mod
34:     go.opentelemetry.io/otel v1.25.0

go.mod
82:     go.opentelemetry.io/otel v1.25.0 // indirect
```

**Comment by sharathsivakumar** [comment]
@ivanvc Thanks for looking into this. It is bumped. Only that, it was bumped by another package. I think the go mod tidy bumped it while upgrading otel/trace.

File: tests/go.mod [hunk 1]
```diff
@@ -36,9 +36,9 @@ require (
 	go.etcd.io/gofail v0.1.0
 	go.etcd.io/raft/v3 v3.6.0-alpha.0
 	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.50.0
-	go.opentelemetry.io/otel v1.25.0
-	go.opentelemetry.io/otel/sdk v1.25.0
-	go.opentelemetry.io/otel/trace v1.25.0
+	go.opentelemetry.io/otel v1.26.0
```
Comment (sharathsivakumar) [inline tests/go.mod:39]: @ivanvc you can check here!

File: server/go.mod [hunk 2]
```diff
@@ -31,9 +31,9 @@ require (
 	go.etcd.io/etcd/pkg/v3 v3.6.0-alpha.0
 	go.etcd.io/raft/v3 v3.6.0-alpha.0
 	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.50.0
-	go.opentelemetry.io/otel v1.25.0
-	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.25.0
-	go.opentelemetry.io/otel/sdk v1.25.0
+	go.opentelemetry.io/otel v1.26.0
```
Comment (sharathsivakumar) [inline server/go.mod:34]: @ivanvc and here!

**Comment by sharathsivakumar** [comment]
@ivanvc I have left comments where it has been bumped. Can you please check and verify it. I think I did not mention this in the MR description. Thats fixed as well.

**Review by ivanvc — APPROVED** [review]
Got it, thanks @sharathsivakumar.
/lgtm

**Comment by ivanvc** [comment]
@ahrtr, @jmhbnz PTAL :)

**Review by ahrtr — APPROVED** [review]
LGTM

Thanks

**Pull request merged**